pasjonsfrukt/main.py

```diff
@@ -99,47 +104,66 @@ def build_podcast_episode_file_path(config: Config, podcast_slug: str, episode_i
     return build_podcast_dir(config, podcast_slug) / f"{episode_id}.mp3"
 
 
-def build_feed(config: Config, episodes: list[podme_api.types.PodMeEpisode], slug: str, title: str, description: str,
-               image_url: str):
+def build_feed(
+        config: Config,
+        episodes: list[PodMeEpisode],
+        slug: str,
+        podcast: PodMePodcast,
+):
     secret_query_param = get_secret_query_parameter(config)
     items = []
     for e in episodes:
-        episode_id = e['id']
+        episode_id = e.id
         episode_path = f"{slug}/{episode_id}"
         items.append(Item(
-            title=e['title'],
-            description=e['description'],
+            title=e.title,
+            description=e.description,
             guid=Guid(episode_id, isPermaLink=False),
             enclosure=Enclosure(
                 url=f'{config.host}/{episode_path}/{secret_query_param}',
                 type='audio/mpeg',
                 length=build_podcast_episode_file_path(config, slug, episode_id).stat().st_size
             ),
-            pubDate=date_of_episode(e),
+            pubDate=e.dateAdded,
             extensions=[
                 iTunesItem(
-                    duration=e['length']
+                    author=e.authorFullName,
+                    duration=e.length,
                 )
             ]
         ))
     feed_link = f"{config.host}/{slug}/{secret_query_param}"
     feed = Feed(
-        title=title,
+        title=podcast.title,
         link=feed_link,
-        description=description,
+        description=podcast.description,
         language="no",
         image=Image(
-            url=image_url,
-            title=title,
-            link=feed_link
+            url=podcast.imageUrl,
+            title=podcast.title,
+            link=feed_link,
         ),
+        categories=[c.key for c in podcast.categories],
```

Comment:
`categories` is `Optional` right?
```suggestion
        categories=[c.key for c in podcast.categories] if podcast.categories is not None else None,
```
Similar checks should be performed for the other optional attributes